hack/jenkins/e2e-runner.sh

```diff
@@ -206,9 +206,11 @@ fi
 
 cd kubernetes
 
-# Upload build start time and k8s version to GCS, but not on PR Jenkins.
-if [[ ! "${JOB_NAME}" =~ -pull- ]]; then
```

Comment:
I guess maybe we still want to avoid running this here on pull requests, instead running it before we build?